core/common/src/main/java/alluxio/heartbeat/HeartbeatContext.java

```diff
@@ -39,6 +39,8 @@
   public static final String MASTER_LOST_MASTER_DETECTION = "Master Lost Master Detection";
   public static final String MASTER_LOST_WORKER_DETECTION = "Master Lost Worker Detection";
   public static final String MASTER_METRICS_SYNC = "Master Metrics Sync";
+  public static final String MASTER_S3A_INTERMEDIATE_UPLOAD_CLEAN
+      = "Master S3A Intermediate Upload Clean";
```

Comment:
I would suggest to make this agnostic to the UFS, ie. `Master UFS Cleanup` and call a `cleanup` method for each of the registered UnderFileSystems which for S3 does the existing logic. @madanadit what do you think about adding `cleanup` to the UFS interface?

Reply:
Create a `UnderFileSystem.cleanup()` method. And setup periodically cleanup all non-readonly ufs in `DefaultFileSystemMaster`
Thanks